[@lit-labs/signals] export `SignalWatcherApi` type

### Which package(s) are affected?

Other/unknown (please mention in description)

### Description

`@lit-labs/signals@0.2.0`

When extending `SignalWatcher`, typescripts throws an error because `SignalWatcherApi` isn't exported:
```
error TS4020: 'extends' clause of exported class 'MyElement' has or is using private name 'SignalWatcherApi'.
```

### Reproduction

https://www.typescriptlang.org/play/?#code/JYWwDg9gTgLgBAbwDLBgUQDYFMRYHYwC+cAZlBCHAOQapUDcAUKJLIgMrADmeAhhgHVeMAMYALLFGJkKcAEQABWjAC0GXgCMAzgHot3Phi1ymjLAA9W8Eeq1a4ncNgDiULFhjA8XOBZj4AE3tOHn4hUQkoAAoUdGxcAgBKREIgA

### Workaround

I have not found a workaround

### Is this a regression?

No or unsure. This never worked, or I haven't tried before.

### Affected versions

Failing in @lit-labs/signals@0.2.0, interface did not exist before

### Browser/OS/Node environment

node 25.2.1
npm 11.6.4
tsc 5.9.3
linux 6.17.9

--- a/.changeset/true-ads-strive.md
+++ b/.changeset/true-ads-strive.md
@@ -1,0 +1,5 @@
+---
+'@lit-labs/signals': minor
+---
+
+export `SignalWatcherApi` interface which is used in the return type of the `SignalWatcher` mixin

--- a/packages/labs/signals/src/lib/signal-watcher.ts
+++ b/packages/labs/signals/src/lib/signal-watcher.ts
@@ -41,7 +41,7 @@ const effectWatcher = new Signal.subtle.Watcher(() => {
   });
 });
 
-interface SignalWatcherApi {
+export interface SignalWatcherApi {
   updateEffect(fn: () => void, options?: EffectOptions): () => void;
 }
 

